src/app/utilities/cookies.js

```diff
@@ -10,7 +10,7 @@ export default class cookies {
         return cookiesObject;
     }
 
-    static add(name, value, path, domain) {
+    static add(name, value, path, domain, sameSite, secure) {
```

Comment:
This method has got 6 `if` statements. Consider breaking out the `error` handling into a separate function as these statements are not "`add`ing" anything (as per method name).